### PR TITLE
POM changes

### PR DESCRIPTION
**Pull request by tsegismont**
Use C* driver and Guava version from parent + mvn tidy:pom

### CODE REVIEW

File: containers/kubernetes/pom.xml [hunk 1]
```diff
@@ -29,9 +29,10 @@
   </parent>
 
   <artifactId>hawkular-metrics-kubernetes-app</artifactId>
-  <name>Hawkular Metrics Kuberenetes Application</name>
   <version>0.3.5-SNAPSHOT</version>
 
+  <name>Hawkular Metrics Kuberenetes Application</name>
```
Comment (pilhuhn) [inline containers/kubernetes/pom.xml]: Typo in K8s

**Comment by pilhuhn** [comment]
Lmgtm  - perhaps @ppalaga to double check?

File: containers/kubernetes/pom.xml (hunk 1 above)
Comment (tsegismont) [inline containers/kubernetes/pom.xml]: Le 17/06/2015 10:35, Heiko W. Rupp a écrit :
> Typo in K8s

Fixed

**Comment by ppalaga** [comment]
I do not see anything suspicious, but note that /me knows little about metrics internals.

**Comment by tsegismont** [comment]
Thank you guys

**Comment by jsanda** [comment]
We need to be careful about the version of Guava we use since it is also a dependency of the C* driver. We have bumped the driver version to 2.1.6, which still uses Guava 14.0.1. I have run into problems in the past by using a different version. Just need to be careful. The same would apply with other driver deps like Netty.

**Comment by tsegismont** [comment]
Le 17/06/2015 13:21, jsanda a écrit :
> We need to be careful about the version of Guava we use since it is also
> a dependency of the C* driver. We have bumped the driver version to
> 2.1.6, which still uses Guava 14.0.1. I have run into problems in the
> past by using a different version. Just need to be careful. The same
> would apply with other driver deps like Netty.

Good point. Note that I haven't changed the effective POMs, only removed 
the Guava version declaration in Metrics, as it was the same as in parent.

**Pull request merged**